user/pages/10.developer-guide/04.development_framework/22.workflow-functions/default.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'How to add expression functions'
+media_order: 'Screenshot from 2023-10-25 10-41-28.png'
+metadata:
+    description: 'How to add expression functions.'
+    author: 'Arlind Ismalaja'
+content:
+    items:
+        - '@self.children'
+    order:
+        by: date
+        dir: desc
+    url_taxonomy_filters: true
+taxonomy:
+    category:
+        - webservice
+        - application
+    tag:
+        - howto
+---
+
```

Comment:
we need a small explanation here followed by three equal signs `===` as you can see here:

https://github.com/coreBOS/coreBOSDocumentation/blob/main/user/pages/05.configuration-tools/05.gendoc/05.howto/item.md?plain=1

that is useful for parsers as it is understood as a description of the contents of the page